yarn-project/aztec-sandbox/src/server.ts

```diff
@@ -1,23 +1,21 @@
-import { createPXERpcServer } from '@aztec/pxe';
-import { PXE } from '@aztec/types';
+import { JsonRpcServer } from '@aztec/foundation/json-rpc/server';
 
 import http from 'http';
 
-import { createApiRouter } from './routes.js';
-
 /**
- * Creates an http server that forwards calls to the PXE and starts it on the given port.
- * @param pxe - PXE that answers queries to the created HTTP server.
+ * Creates an http server that forwards calls to the underlying instance and starts it on the given port.
+ * @param instance - Instance to wrap in a JSON-RPC server.
  * @param port - Port to listen in.
```

Comment:
seems this is missing `@param jsonRpcFactoryFunc`

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/2492/commits/5aed56c6badbda440fcde2815df63f0e366f58ea